cluster-api/src/main/java/com/sequenceiq/cloudbreak/cluster/api/ClusterModificationService.java

```diff
@@ -93,4 +93,12 @@ default void restartAll(boolean withMgmtServices) {
     default void restartClusterServices() {
         throw new UnsupportedOperationException("Interface not implemented.");
     }
+
+    void stopClouderaManagerService(String serviceType)  throws Exception;
+
+    void startClouderaManagerService(String serviceType)  throws Exception;
+
+    Map<String, String> readServices()  throws Exception;
+
+    void updateServiceConfig(String serviceName, Map<String, String> config, List<String> roleGroupNames) throws CloudbreakException;
```

Comment:
Once vertical scaling is done, it will survive upgrade/repair/upscaling. I have tested out upscaling and upgrade, but not the repair. but following logic, ephemeral disks are mounted on ephfs paths after repair and it should work without issues for the compute nodes.